src/main/flags.sh

```diff
@@ -154,7 +154,7 @@ else
             "-lbase_stubs"
             "-lbigstringaf_stubs"
             "-lca_certs_stubs"
-            "-lcamlstr"
+            #"-lcamlstr"
```

Comment:
why is this commented out now?

Reply:
on purpose, to generate errors in CI one can inspect, to decide whether or not the current error output is enough to be directed towards src/main/flags.sh

I'll remove the '#' once this PR is approved.

Reply:
ahh ok